OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseOutlineFragment.java

```diff
@@ -91,9 +105,101 @@ public void onItemClick(AdapterView<?> parent, View view, int position, long id)
             }
         });
 
+        // We only want long-press-to-delete functionality in only videos mode
+        if (isVideoMode) {
+            listView.setOnItemLongClickListener(new AdapterView.OnItemLongClickListener() {
+                @Override
+                public boolean onItemLongClick(AdapterView<?> parent, View view, int position, long id) {
+                    if (((IconImageView) view.findViewById(R.id.bulk_download)).getIcon() == FontAwesomeIcons.fa_check) {
```

Comment:
Yes, that is another way. But I found this way easier since the listener is already giving us the actual view at that position.